### PR TITLE
Update Spanish Readme

### PR DESCRIPTION
**Pull request by Josexv1**
Open to review.

### CODE REVIEW

**Comment by Josexv1** [comment]
@anonimal Updated, check it.

File: i18n/es/README.md [hunk 1]
```diff
@@ -37,100 +37,100 @@ Alpha release coming soon
...
-## Vulnerability Response
-- Our [Vulnerability Response Process](https://github.com/monero-project/meta/blob/master/VULNERABILITY_RESPONSE_PROCESS.md) encourages responsible disclosure
-- We are also available via [HackerOne](https://hackerone.com/monero)
-
-## Contact and Support
-- IRC: [Freenode](https://webchat.freenode.net/) | Irc2P with Kovri
-  - `#kovri` | Community & Support Channel
-  - `#kovri-dev` | Development Channel
+## Documentación y desarrollo
+- Una [Guía de usuario](https://github.com/monero-project/kovri-docs/blob/master/i18n/en/user_guide.md) esta disponible para todos los usuarios
```
Comment (ordtrogen) [inline i18n/es/README.md]: está

File: i18n/es/README.md [hunk 2]
```diff
@@ -37,100 +37,100 @@ Alpha release coming soon
...
+- Una [Guía de desarrolladores](https://github.com/monero-project/kovri-docs/blob/master/i18n/en/developer_guide.md) esta disponible para los desarrolladores (por favor, leer antes de abrir un nuevo pull request)
```
Comment (ordtrogen) [inline i18n/es/README.md]: está

File: i18n/es/README.md [hunk 3]
```diff
@@ -37,100 +37,100 @@ Alpha release coming soon
...
-  - All other contact
+  - Otros contactos
     - anonimal [at] getmonero.org
-    - PGP Key fingerprint: 1218 6272 CD48 E253 9E2D  D29B 66A7 6ECF 9144 09F1
+    - Huella de PGP: 1218 6272 CD48 E253 9E2D  D29B 66A7 6ECF 9144 09F1
 
-## Donations
-- Visit our [Donations Page](https://getmonero.org/getting-started/donate/) to help Kovri with your donations
+## Donaciones
+- Visita nuestra [Pagina de Donaciones](https://getmonero.org/getting-started/donate/) para ayudar a Kovri con tus donaciones
```
Comment (ordtrogen) [inline i18n/es/README.md]: Página

File: i18n/es/README.md [hunk 4]
```diff
@@ -37,100 +37,100 @@ Alpha release coming soon
...
 | Graphviz            | 2.36                         |    X     | graphviz    | graphviz         | graphviz         | graphviz      | graphviz    |
-| Docker              | Always latest stable version |    X     | See website | See website      | See website      | See website   | See website |
+| Docker              | Siempre la última versión estable |    X     | Ver sitio web | Ver sitio web      | Ver sitio web      | Ver sitio web   | Ver sitio web |
 
 #### Windows (MSYS2/MinGW-64)
-* Download the [MSYS2 installer](http://msys2.github.io/), 64-bit or 32-bit as needed
-* Use the shortcut associated with your architecture to launch the MSYS2 environment. On 64-bit systems that would be the `MinGW-w64 Win64 Shell` shortcut. Note: if you are running 64-bit Windows, you'll have both 64-bit and 32-bit environments
-* Update the packages in your MSYS2 install:
+* Descarga el instalador de [MSYS2](http://msys2.github.io/), 64-bit o 32-bit
+* Usa el acceso directo asociado con tu arquitectura, para abrir el entorno MSYS2. En sistemas 64-bit  seria `MinGW-w64 Win64 Shell`. Nota: si estas en 64-bit Windows, vas a necesitar ambos entornos 64-bit y 32-bit
```
[the remark this answers is not in the document]
Comment (ordtrogen) [reply i18n/es/README.md]: si estás en 64-bit

**Comment by Josexv1** [comment]
Updated with the changes @ordtrogen suggested. Ready to merge.

**Comment by ordtrogen** [comment]
You're doing great work! Hope you don't mind me jumping in and picking on details  :-)

**Comment by Josexv1** [comment]
Keep doing it, normally I just use the Firefox Spellcheck, since my OS is in Japanese and i don't even have a physical ñ, I use this [website ](http://www.notengoenie.com/) to fix everything.

**Comment by Josexv1** [comment]
@anonimal This is ready to merge.

**Pull request merged**